hotfixes/20170804_remove_duplicate_minor_versions.yml

```diff
@@ -0,0 +1,17 @@
+---
+# Used to fix multiple copies of the same minor versions on QA and TEA when
+# more than one page was published while the book was baking.
+
+- name: fix multiple copies of the same minor versions on QA and TEA
+  hosts: publishing
+  tasks:
+    - name: download python script to remove duplicate minor versions
+      get_url:
+        url: "https://raw.githubusercontent.com/Connexions/devops/master/scripts/remove_duplicate_minor_version_books.py"
+        dest: "/tmp"
+
+    - name: run python script to remove duplicate minor versions
+      shell: "/var/cnx/venvs/publishing/bin/python /tmp/remove_duplicate_minor_version_books.py -d 'dbname={{ archive_db_name }} user={{ archive_db_user }} password={{ archive_db_password }} host={{ archive_db_host }} port={{ archive_db_port }}'"
+
+    - name: disable update_default_modules_stateid trigger
```

Comment:
Now that I think about it, we should probably run this on `database` instead of `publishing`.  In case `psql` is not installed on `publishing`...